addons/point_of_sale/data/point_of_sale_data.xml

```diff
@@ -54,16 +54,6 @@
             <field name="is_pos_groupable">True</field>
         </record>
 
-        <record model="pos.bill" id="0_01" forcecreate="0">
-            <field name="name">0.01</field>
-            <field name="value">0.01</field>
-        </record>
-
-        <record model="pos.bill" id="0_02" forcecreate="0">
-            <field name="name">0.02</field>
-            <field name="value">0.02</field>
-        </record>
-
```

Comment:
why are you removing these? keep it. remove it in master if you want.

Reply:
Hello @caburj 
We were trying to merge both tasks: 3728728, 3727068

But now separate PR will take care of the second task - 3727068

Thanks!